src/main/webapp/style.css

```diff
@@ -1,3 +1,10 @@
+@font-face {
```

Comment:
Before we keep adding to this style.css and make it really big, lets split this up. Please do this in a separate PR.